datadog/resource_datadog_security_monitoring_default_rule.go

```diff
@@ -100,6 +100,13 @@ func resourceDatadogSecurityMonitoringDefaultRule() *schema.Resource {
 					Computed:    true,
 					Description: "The rule type.",
 				},
+
+				"tags": {
+					Type:        schema.TypeSet,
+					Optional:    true,
+					Description: "Tags for generated signals.",
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Could you add the fact that: `Tags must contain all OOTB rule tags`.‏

Reply:
Added to the description. One thing I realized is that anyone upgrading to this version of our terraform provider will have to import all these tags initially, and they'll also have to import them any time we change the tags on a default rule. I just want to confirm that's expected and there's no better way to do this?

Reply:
Yes, this API is working like that.

Reply:
Good point.
You are right, it could be an issue for rule update.
I think we should append to existing ones (by doing a get before doing the put).